wagn/lib/wagn/commands.rb

```diff
@@ -9,15 +9,15 @@ def load_rake_tasks
 
 RAILS_COMMANDS = %w( generate destroy plugin benchmarker profiler console server dbconsole application runner )
 ALIAS = {
-  "rs" => "rspec",
-  "cc" => "cucumber",
-  "jm" => "jasmine",
-  "g"  => "generate",
-  "d"  => "destroy",
-  "c"  => "console",
-  "s"  => "server",
-  "db" => "dbconsole",
-  "r"  => "runner"
+  'rs' => 'rspec',
+  'cc' => 'cucumber',
+  'jm' => 'jasmine',
+  'g'  => 'generate',
+  'd'  => 'destroy',
+  'c'  => 'console',
+  's'  => 'server',
+  'db' => 'dbconsole',
+  'r'  => 'runner'
```

Comment:
yeah, I like for us to do this where we can; thanks for doing it here.  (I also like that single quotes are precisely half as visually noisy!)